index.html

```diff
@@ -2,10 +2,96 @@
 <html>
     <head>
         <title>Tristan Geary</title>
+		
+		<!-- Metadata -->
+		<meta name= "author" content = "Tristan Geary">
+		<meta name= "description" content = "Lab2 webpage assignment">
+		<meta charset= "UTF-8">
+		<meta name="keywords" content= "html">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+		
+		
     </head>
+	
     <body>
-        <h1>Larger Header</h1>
-        <h4>SMALL HEADER</h4>
-        <p>one paragraph for CS203 - Tristan Geary.</p>
+		
+		<!-- academic information as headers -->
+		<div>
+		<h1><b>Tristan Geary: Undergraduate Student</h1>
+		
+		<!-- horizontal ruler -->
+		<hr>
+		
+        <h2>Major: Neuroscience</h2>
+		
+		<h2>Minor: Computer Science </h2>
+		</div>
+		
+		<hr>
+		
+		<div>
+		<!-- fun fact -->
+		<p>Fun Fact:I was originally a biology and business student, but <br> switched
+		because it didn't align with my goals<br>
+		<div/>
+		
+		<hr>
+        
+		<!-- career goals section -->
+		<div>
+		<h3><em>Career Goals:</em></h3>
+		
+		<!-- unordered list -->
+		<ul>
+		<li><p>I hope to go to <strong> medical school </strong> and become a <br> <b>doctor
+		(I am currently applying to medical schools this semester).</p></li>
+		<li><p>I hope to do a lot of medically-related research.</p></li>
+		<li><p>Currently, I want to become a <b>surgeon.</p></li>
+		</ul>
+		
+		</div>
+		<hr>
+		
+		<!-- table detailing schools attended, favorite classes, and year graduated -->
+		<div>
+		<h2>Schools attended, favorite class, and year graduated:</div>
+        <table border="1", style = "width:50%">
```

Comment:
There are no `,` between attributes, only spaces.

Also, border should be inside the style:
`<table style="width:50%; border: 1px">`